doppler/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e22c8861-f652-42a9-9582-6470504b421f",
+  "app_id": "doppler",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Doppler Secrets Management",
+    "title": "Doppler",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Organize secrets into projects",
+        "image_url": "images/projects.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Configure secret visibility to prevent accidental access",
+        "image_url": "images/secret-visibility.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Provide existing secrets or generate cryptographically random values",
+        "image_url": "images/generate.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Set up automatic rotation",
+        "image_url": "images/rotation.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Allow individual developers to override values in personal configs",
+        "image_url": "images/personal-config.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Track all secret reads and writes",
+        "image_url": "images/version-history.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Analyze activity patterns",
+        "image_url": "images/dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Security",
+      "Category::Developer Tools",
+      "Category::Log Collection",
+      "Offering::Integration"
```

Comment:
this is missing the submitted data type. Please add this https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags

Reply:
I've added the logs datatype